docs/news/HEAD.rst

```diff
@@ -20,16 +16,260 @@ New and extended options
+
 New parsers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The following parsers have been added:
 
-* V
-* Quarto *Markdown based subparser*
-* BibLaTeX *BibTeX based subparser*
-* Forth *optlib*
-* Terraform (HCL) *optlib*
-* TerraformVariables *optlib*
+* BibLaTeX *BibTeX based subparser* by masatake · Pull Request #3822
+* Forth *optlib* by farvardin · Pull Request #3812
+* Quarto: new parser by masatake · Pull Request #3677
+
+  +  Quarto by Anish Shah · Pull Request #3643
+
+* V: new parser by Tim Marston · Pull Request #3871, #3870
+* Terraform: new parser by masatake · Pull Request #3684
+
+  + Terraform (HCL) (*.tf): new parser by  Matt Mrowiec · Pull Request #3683
+  + Terraform (HCL) (*.tf): new parser by Antony Southworth · Pull Request #2952
+
+* TerraformVariables *optlib* by masatake · Pull Request #3684
+
+Changes about parser specific kinds, roles, fields, and extras
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. See the output of ./misc/news.bash man [v6.0.0]
+
+* AutoIt
+
+  + Drop `$` from tags for variables names.
+
+* Automake
+
+  + New extra `canonicalizedName`
+  + New kind `pseudodir`
+
+* C
+
+  + New role `foreigndecl` for `function` kind
+  + New role `foreigndecl` for `struct` kind
+
+* Fortran
+
+  + New extra `linkName`.
+
+* Kconfig
+
+  + New kind `variable`
+
+* JavaScript
+
+  + New role `foreigndecl` for `function` kind
+
+* Markdown
+
+  + New kind  `hashtag`
+
+* SystemTap
+
+  + New role `attached` for `probe` kind
+
+* SystemVerilog
+
+  + New kind `define`
 
 Readtags
 ---------------------------------------------------------------------
+* Enhancement: add `-A, --absolute-input` and `-C, --canonicalize-input` options.
+  See :ref:`readtags(1) <readtags(1)>`. for more details.
+* Enhancement: add `-d, --debug`  option for debugging purpose.
+* Bugfix: make `--formatter` options works.
+
+
+Merged pull requests
+---------------------------------------------------------------------
+
+.. note::
+
+   This list is imperfect. masatake cleaned up some pull requests before
+   merging. Though his names is used in "... by ...", his is not the
+   primary contributor of the pull requests. See git log for more
+   defatils.
+
+.. generated by ./misc/news.bash pr [v6.0.0]
+
+* docs(web): manage versions of NEWS by masatake · Pull Request #3872
+* main: use the interval tree for filling scope field by masatake · Pull Request #3678
+* V for merging by masatake · Pull Request #3871
+* YACC: fix a typo in the pattern for skipping C strings by masatake · Pull Request #3869
+* Revise: the way of accessing the optVm's appData by masatake · Pull Request #3868
+* dsl: extend #/../ operator to be able to extract a matched group in the pattern by masatake · Pull Request #3768
+* Docs: minor fixes by masatake · Pull Request #3867
+* misc/units.py: fix invalud escape sequences in regex patterns by masatake · Pull Request #3866
```

Comment:
I found some typos in this list.
It seems that this list is generated from the title of the MRs, so I've fixed some titles.

Reply:
Thank you!